src/DotVVM.Samples.Common/Views/ComplexSamples/SPAErrorReporting/site.dotmaster

```diff
@@ -0,0 +1,34 @@
+﻿@viewModel DotVVM.Samples.BasicSamples.ViewModels.ComplexSamples.SPA.SiteViewModel, DotVVM.Samples.Common
+
+<!DOCTYPE html>
+
+<html lang="en" xmlns="http://www.w3.org/1999/xhtml">
+<head>
+    <meta charset="utf-8" />
+    <title></title>
+</head>
+<body>
+
+    <h1>SPA Test</h1>
+
+    <dot:RouteLink RouteName="ComplexSamples_SPAErrorReporting_default" Text="Default Page" />
+    <dot:RouteLink RouteName="ComplexSamples_SPAErrorReporting_test" Text="Test Page" />
+
+    <dot:SpaContentPlaceHolder ID="Container" />
+
+    <div>Number of logged errors: <span ID="numberOfErrors">0</span></div>
+
+    <dot:InlineScript Dependencies="dotvvm">
+        var numberOfErrors = 0;
+        dotvvm.events.error.subscribe(
+            function (error) {
+                var element = document.getElementById('numberOfErrors');
+                element.innerText = ++numberOfErrors;
```

Comment:
I don't understand how this test works. When does the error occur? Does the UI test anything?

Reply:
The issue that is fixed by this PR caused the dotvvm.events.error not be called when app was offline and the SPA navigation (or any other server connection). This UI only demonstrates that the event is called properly when you turn offline mode in the development tools of your browser. This page was supposed to be used as part of UI test, but I did not find a way to make the selenium driver change the page to think its offline. So I keeped it for manual testing. 

I can remove it if you want.

Reply:
No, that's fine. Could you please just add this explanation to the test, so that we can actually test it sometime in the future?

Reply:
Done.